interface/src/app/analyze/analysis/analysis.less

```diff
@@ -1,3 +1,37 @@
+/*
+ * styles to make sampleBin editing controls more user-friendly
+ */
+.row-control {
+  float: none;
+  .opacity(.2);
+
+  &:hover,
+  &:focus {
+    text-decoration: none;
+    cursor: pointer;
+    .opacity(.5);
+  }
+
+  button& {
+    padding: 0;
+    cursor: pointer;
+    background: transparent;
+    border: 0;
+    -webkit-appearance: none;
+  }
+}
+// allow us to use controls to help align headers, but don't show them
+th .row-control, thead .row-control {
+  visibility: hidden;
+}
+
+// sample group highlighting
```

Comment:
Turns out that constants like this belong in `src/less/variables.less`, so I moved them there. I also put a better comment here. There's more cleanup in the `.less/css` department to tackle along with issue #76.